src/pages/guides/configuration/index.md

```diff
@@ -371,6 +371,22 @@ async function main (params) {
 exports.main = main
 ```
 
+As an alternative, environment variables in .env can be declared as inputs to a package, to make them available to all the actions in the package via their parameters.
```

Comment:
I think that package-level parameters can still be overridden on action level. This should be added imo.

Reply:
Good point!
I've rewritten the statement a bit and noted the option to override at action level:
```
Alternatively, environment variables defined in a .env file can be configured as inputs to a package, making them accessible to all actions within the package through their parameters. 
Keep in mind that package-level inputs can still be overridden at the action level.
```